Commands.md

```diff
@@ -1628,3 +1630,21 @@ Commits changes with a generated message.
 ## git continue
 
 Continue current revert, rebase, merge or cherry-pick, without the need to find exact command in history.
+
+## git wip
+
+Create a Work In Progress(WIP) commit, which will include all changes in the
+working directory. (i.e., changes to existing files, new files, removed files)
+
+```bash
+$ git wip
+```
+
+## git unwip
+
+Undo A Work In Progress(WIP) commit and put all of those changes back into the
```

Comment:
```suggestion
Undo a Work In Progress(WIP) commit and put all of those changes back into the
```

Reply:
I have updated the commit with this change.